test/utils.test.sh

```diff
@@ -321,6 +321,24 @@ log="$(exact_match)"
 assert "equal" "$FAILURE" ""
 assert "contains" "$log" ""
 
+# ecr_cleanup()
+tag_test "ecr_cleanup"
+test_region="us-east-1"
+test_repo="test-repo"
+
+function cleanup_filepath() {
```

Comment:
Yeah since you're actually calling an external file, you can't just mask the bash function in order to mock this call. This test is working now because of the other missing `$` I noted, which would actually break at runtime. 

Instead you need to do something like:

- make `cleanup.js` live somewhere in `$PATH` when conex is running, perhaps symlinked as `cleanup_ecr`
- make the call to this script `cleanup_ecr` instead of `./path/to/cleanup.js`
- in your test, you can mask that `cleanup_ecr` command. I'm not 100% sure if a function will work, or if you'll need to use a alias, or put an actual dummy file in `$PATH` before the real `cleanup_ecr`